package.json

```diff
@@ -80,6 +80,9 @@
     "@babel/plugin-proposal-class-properties": "^7.18.6",
     "@cocos/box2d": "1.0.1",
     "@cocos/cannon": "1.2.8",
+    "@cocos/ccbuild": "2.2.9",
```

Comment:
There is `"@cocos/ccbuild": "2.2.10",` below. Is this duplicated?

Reply:
`"@cocos/dragonbones-js": "^1.0.1"`, is also duplicated.

Reply:
The changes brought in during the code merge.

Reply:
Modified